CI/travis.linux.install.sh

```diff
@@ -0,0 +1,5 @@
+#/bin/bash
```

Comment:
That's totally the way it should be - if we don't need something, we remove it, and if we need it again, we add it back. Keeps the codebase clean from cruft.